plans/dht/test/provide_stress.go

```diff
@@ -0,0 +1,70 @@
+package test
+
+import (
+	"github.com/ipfs/testground/sdk/runtime"
+)
+
+// TODO this entire test needs to be revisited.
```

Comment:
Can you unpack this? What needs to be achieved (so that I or someone else can tackle it)